cmds/webboot/webboot.go

```diff
@@ -97,9 +97,16 @@ func main() {
 	}
 
 	cachedDir := *dir
+
 	if cachedDir == "" {
-		// todo: find the cache directory by check block devices
-		log.Fatal("Please provide a valid dir argument")
+		mp, err := getCachedDirectory()
+		if err != nil {
+			log.Fatalf("Fail to find the USB stick: %+v", err)
+		}
+		if mp == nil {
```

Comment:
getCachedDirectory should always return nil if there is an error.

Reply:
do you mean I should not log.Fatal in the getCachedDirectory function?